W/wget/build_tarballs.jl

```diff
@@ -41,8 +34,12 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency("GnuTLS_jll"),
-    Dependency("Nettle_jll", v"3.7.2"; compat="~3.7.2"),
+    Dependency(PackageSpec(name="Libuuid_jll", uuid="38a345b3-de98-5d2b-a5d3-14cd9215e700"); compat="2.40.2"),
+    Dependency(PackageSpec(name="OpenSSL_jll", uuid="458c3c95-2e84-50aa-8efc-19380b2a3a95"); compat="3.0.15"),
+    Dependency(PackageSpec(name="PCRE2_jll", uuid="efcefdf7-47ab-520b-bdef-62a2eaa19f15"); compat="10.43.0"),
```

Comment:
https://github.com/JuliaLang/julia/blob/f9720dc2ebd6cd9e3086365f281e62506444ef37/stdlib/PCRE2_jll/Project.toml#L3

Reply:
I removed the compat entry for `PCRE2`.